fmriprep/__about__.py

```diff
@@ -89,14 +89,14 @@
     'nilearn!=0.5.0,!=0.5.1',
     'nipype>=1.1.6',
     'nitime',
-    'niworkflows',
+    'niworkflows>=0.9',
     'numpy',
     'pandas',
     'psutil>=5.4',
     'pybids<0.8.0a0,>=0.7.1',
     'pyyaml',
     'scikit-image',
-    'smriprep',
+    'smriprep>=0.2.0a',
```

Comment:
version pinning here disables reading dependency links.

Reply:
Those dependency links do nothing anyway. Have you ever found them to work?

Reply:
They are definitely finicky. But the version here will likely remove the version you've previously installed via `requirements.txt`.

Then, when it gets to process the dependency links, I think setuptools has a hard time figuring out the version of the commit (because that is returned by versioneer, and that needs to have access to tags in the repo).

But yeah, I can't definitely say yes to your question.